sdl_android/src/main/java/com/smartdevicelink/transport/RouterServiceValidator.java

```diff
@@ -345,22 +345,22 @@ protected boolean verifyVersion(int version, JSONArray versions){
 	/**
 	 * Using the knowledge that all SDL enabled apps have an SDL Broadcast Receiver that has an intent filter that includes a specific 
 	 * intent. 
-	 * @return 
+	 * @return the list of Sdl enabled apps if any app is found; null or empty list otherwise
 	 */
-	private static List<SdlApp> findAllSdlApps(Context context){
+	protected static List<SdlApp> findAllSdlApps(Context context){
 		List<SdlApp> apps = new ArrayList<SdlApp>();
 		PackageManager packageManager = context.getPackageManager();
 		Intent intent = new Intent();
 		intent.setAction("sdl.router.startservice");
```

Comment:
Since there are already changes here I think we should change this from the hardcoded string